cmd/neofs-lens/internal/storage/root.go

```diff
@@ -210,6 +206,18 @@ func openEngine() (*engine.StorageEngine, error) {
 			}
 		}
 
+		var writeCacheOpts []writecache.Option
+		if wcRead := shCfg.WritecacheCfg; wcRead.Enabled {
+			writeCacheOpts = append(writeCacheOpts,
+				writecache.WithPath(wcRead.Path),
+				writecache.WithMaxObjectSize(wcRead.MaxObjSize),
+				writecache.WithMaxCacheSize(wcRead.SizeLimit),
+				writecache.WithNoSync(wcRead.NoSync),
+				writecache.WithMaxFlushBatchDelay(wcMaxBatchDelay),
```

Comment:
1. Delay can be different for WC.
2. Count/size/thresh are all important for proper batches, batch size is not the only parameter.